common/src/main/java/io/netty/util/internal/MathUtil.java

```diff
@@ -65,6 +65,20 @@ public static boolean isOutOfBounds(int index, int length, int capacity) {
     }
 
     /**
+     * Compares two {@code int} values.
+     *
+     * @param  x the first {@code int} to compare
+     * @param  y the second {@code int} to compare
+     * @return the value {@code 0} if {@code x == y};
+     *         {@code -1} if {@code x < y}; and
+     *         {@code 1} if {@code x > y}
+     */
+    public static int compare(int x, int y) {
```

Comment:
No, it's 1.6 in the [pom.xml](https://github.com/netty/netty/blob/c92565d5c7000d2bd74355fe18cbdf98bfc5ba0d/pom.xml#L625).